CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.8.0-RC
+
+* Implement the library for the Web Assembly (Wasm) (#3713). Thanks @igoriakovlev!
```

Comment:
I think it's better to mention that only wasm-js is supported (as f.e. kotlin.serialization supports both wasm-js and wasm-wasi)